.tours/2---object agnostic design.tour

```diff
@@ -0,0 +1,25 @@
+{
+    "title": "2 - Object-Agnostic Design",
+    "steps": [
+        {
+            "file": "es-base-code/main/default/objects/Customer_Fields__mdt/fields/Sobject_Type__c.field-meta.xml",
+            "description": "The app uses Custom Metadata Customer Fields to allow administrators to configure Lightning Web Components and Flow Screen Components to work with any object",
```

Comment:
Use Markdown formatting like Bold, Code etc to differentiate between name of the metadata type and the remaining text.

Reply:
And may be go a lil deep by saying using this metadata type, we configure so and so things.

Reply:
done!